src/status_im/ui/screens/chat/message/gap.cljs

```diff
@@ -17,18 +17,19 @@
                                 gap-ids
                                 chat-id]
                   connected?   [:mailserver/connected?]
+                  use-status-nodes? [:mailserver/use-status-nodes?]
                   first-gap?   (= gap-ids #{:first-gap})]
     (when (or (not first-gap?) public? community?)
       [react/view {:style (style/gap-container)}
        [react/touchable-highlight
-        {:on-press (when (and connected? (not in-progress?))
+        {:on-press (when (and connected? (and (not in-progress?) use-status-nodes?))
```

Comment:
I think you can simplify, as `and` is a multi-variadic function, so you can do:

```
(and
  connected?
  (not in-progress?)
  use-status-nodes?)
```

Which should be equivalent to:

```
 (and
    connected?
    (and
       (not in-progress?)
       use-status-nodes?))
```